src/backend/src/prisma/seed-data/teams.seed.ts

```diff
@@ -80,14 +80,33 @@ const huskies = (headId: number): Prisma.TeamCreateArgs => {
   };
 };
 
+const ravensFootball = (headId: number): Prisma.TeamCreateArgs => {
```

Comment:
correct me if im wrong but you added this team initially. since were just adding players to the existing 'ravens' team, you should just get rid of all instances of 'ravensFootball'